Source/Core/DolphinQt/Config/Mapping/MappingIndicator.cpp

```diff
@@ -105,14 +105,16 @@ void MappingIndicator::AdjustGateColor(QColor* color)
     color->setHsvF(color->hueF(), color->saturationF(), 1 - color->valueF());
 }
 
-MappingIndicator::MappingIndicator(ControllerEmu::ControlGroup* group) : m_group(group)
+MappingIndicator::MappingIndicator()
 {
   // TODO: Make these magic numbers less ugly.
-  int required_height = 106;
-
-  if (group && ControllerEmu::GroupType::MixedTriggers == group->type)
-    required_height = 64 + 1;
+  const int required_height = 106;
```

Comment:
Any reason you don't make this (and the ones further below) a `constexpr`?

Reply:
I suppose I will. :)